src/style/app.module.css

```diff
@@ -944,6 +930,96 @@ hr {
   font-size: small;
 }
 
+.justifyspOrganizationEvents {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 20px;
+}
+
+.titlemodalOrganizationEvents {
+  color: #707070;
+  font-weight: 600;
+  font-size: 20px;
+  margin-bottom: 20px;
+  padding-bottom: 5px;
+  border-bottom: 3px solid #eaebef;
```

Comment:
Please change this to a variable. The color of something similar may already be used